README.rst

```diff
@@ -135,41 +134,26 @@ Install Tendrl commons and node\_agent
 Configuration
 -------------
 
-#. Create the tendrl configuration file ``/etc/tendrl/tendrl.conf``.
+#. Create the tendrl configuration file ``/etc/tendrl/node-agent/node-agent.conf.yaml``.
 
    ::
 
-       $ cp commons/etc/tendrl/tendrl.conf.sample /etc/tendrl/tendrl.conf
+       $ cp etc/tendrl/node-agent/node-agent-dev.conf.yaml /etc/tendrl/node-agent/node-agent.conf.yaml
 
    * Configure the following ``etcd_port`` and ``etcd_connection``
-     directives in ``/etc/tendrl/tendrl.conf`` to point to the etcd
+     directives in ``/etc/tendrl/node-agent/node-agent.conf.yaml`` to point to the etcd
      instance discussed in the first step.
 
-#. Install the commons logging configuration file
-   ``/etc/tendrl/commons_logging.yaml``.
-
-   ::
-
-       $ cp commons/etc/samples/logging.yaml.timedrotation.sample \
-         /etc/tendrl/commons_logging.yaml
-
-   .. note::
-
-       There are other sample configuration files in the
-       ``commons/etc/samples`` directory which could be used to setup
-       logging for different system configuration such as via syslog and
-       journald.
-
 #. Install the node agent logging configuration file
    ``/etc/tendrl/node-agent_logging.yaml``.
 
    ::
 
-       $ cp node-agent/etc/logging.yaml.timedrotation.sample \
+       $ cp node-agent/etc/tendrl/node-agent/logging.yaml.timedrotation.sample \
          /etc/tendrl/node-agent_logging.yaml
-       $ cp ceph_integration/etc/logging.yaml.timedrotation.sample \
+       $ cp ceph_integration/etc/tendrl/ceph-integration/logging.yaml.timedrotation.sample \
```

Comment:
What is this and from where it is copied from(ceph-integration, gluster-integration)
If remember correctly, the logging conf for ceph and gluster are created during import and hence this step is not required. Please check that out